amplify/backend/backend-config.json

```diff
@@ -7,6 +7,10 @@
     "bioMoleculesDB": {
       "service": "DynamoDB",
       "providerPlugin": "awscloudformation"
+    },
+    "psvpatents3": {
```

Comment:
we will have to revert this later since this bucket will not trigger the processing

Reply:
Importing the storage bucket will cause a lot of headless errors, the easiest workaround is to create a new bucket. We can chat offline about how to approach a fix for this whether it be reverting or changing a number of other things